packages/modules/payment/src/schema/index.ts

```diff
@@ -0,0 +1,111 @@
+export default `
+enum PaymentCollectionStatus {
+  not_paid
+  awaiting
+  authorized
+  partially_authorized
+  canceled
+}
+
+enum PaymentSessionStatus {
+  authorized
```

Comment:
Should we remove these manually defined GQL schema?

Reply:
It was to have a more concrete test, though, we can keep them for now and rm them when they are migrated to dml

Reply:
But that means we will have to maintain them with the changes done inside the entities as long as we haven't fully migrated

Reply:
Yes thats true. I let you decide 👍

Reply:
Cool. Let's wait and see what @carlos-r-l-rodrigues thinks

Reply:
I believe we should keep them to have more data to work with and test things in more realistic scenario.
But I agree, if schemas are changed we have one more place to update them.

Reply:
Cool. Let's keep them for now